packages/api/src/interfaces/http/Authentification.http.ts

```diff
@@ -48,7 +48,7 @@ export class AuthentificationHttp extends Controller {
         if (req.user) {
             // Successfully logged
             req.res?.cookie("token", req.user.jwt.token, {
-                secure: true,
+                secure: process.env.NODE_ENV === "production" ? true : false,
```

Comment:
```suggestion
                secure: DEV ? false : true,
```
avec `DEV` importé de `env.conf.ts` qui revient à l'opposé de ton test sur `NODE_ENV`. Et de façon générale on essaie de ne pas appeler `process.env` n'importe où et de passer par ces fichiers `conf.ts`

Reply:
oui bien vu 

d'ailleurs on a un peu un doublon sur le fichier `env.conf.ts`

avec 
```js
export const ENV = process.env.ENV || "dev";
export const DEV = process.env.NODE_ENV !== "production";
```

Reply:
tout comme on a un doublon côté scalingo en fait; mais historiquement ça doit avoir une raison (scalingo utilise peut-être que NODE_ENV par ex)